dockstore-webservice/src/main/resources/openapi3/openapi.yaml

```diff
@@ -4310,6 +4310,25 @@ paths:
       - bearer: []
       tags:
       - users
+  /users/getAllUserEmails:
+    get:
+      description: Get the emails of all Dockstore users
```

Comment:
Not 100% certain what our procedure is but I believe all auth/curator-only endpoints are explicitly mentioned